hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncConfig.java

```diff
@@ -123,6 +123,9 @@
   @Parameter(names = {"--conditional-sync"}, description = "If true, only sync on conditions like schema change or partition change.")
   public Boolean isConditionalSync = false;
 
+  @Parameter(names = {"--sync-all-partitions"}, description = "If true, sync all partitions")
```

Comment:
this is a force sync? if so may be make it clearer? `--force-sync-all-partitions`

Reply:
yes, will change it